src/org/sosy_lab/common/NativeLibraries.java

```diff
@@ -95,8 +98,11 @@ public static OS guessOperatingSystem() {
 
   @Deprecated // will become private
   public enum Architecture {
+    // We only consider the mainly used architectures and ignore older ones like PPC or MIPS.
     X86,
-    X86_64;
+    X86_64,
+    ARM,
```

Comment:
I guess this could be misunderstood as "any ARM architecture". Maybe we should use `ARM32`?

Or is it the case like for `x86` that everyone uses `ARM` to refer to 32-bit ARM? Then we would want to keep to the standard. What do compilers use here, for example?

Reply:
I will remove plain `ARM`, as matching for ARM 6 or 7 is complex and might not be required by users.